src/abstractdataframe/abstractdataframe.jl

```diff
@@ -438,8 +438,7 @@ completecases(df)
 ```
 
 """
-function completecases(df::AbstractDataFrame)
-    ## Returns a Vector{Bool} of indexes of complete cases (rows with no NA's).
+function completecases(df::AbstractDataFrame)::AbstractVector{Bool}
```

Comment:
Maybe the type annotation should be added to `res` inside the function instead? We don't want an internal type instability to hurt performance.